### DISCUSSION TITLE
How to disable auto-update that prevents us from editing code?

### DISCUSSION THREAD

**Discussion by OnkelTem** [codesandbox/codesandbox-client#8462] [Q&A]
I don't know what has happened with Codesandbox, but it's barely possible to edit code now. 

I have the preview open. Whenever I type something, it is trying to autoupdate itself taking away the focus so that I cannot type anymore. I click in the editor, getting my focus back, but when I type a letter or two, the Preview autoupdates again stealing the focus one more time!

I tried to disable this annoying and unreasonable behavor, but I didn't find anything related netither in the settings, nor in the user preferences or in the preview panel itself. 100% it worked before. I remember I switched it off. There is simply no need in updating Preview while you type something.

**Comment by filipelima18** [comment]
Hi @OnkelTem. The most common cause for previews hijacking focus is when they contain input fields that take focus. Can you confirm if this is the case for you and if it happens only on one Sandbox or in multiple ones?

**Reply by OnkelTem** [reply]
Sorry for the delay. Yes, I'm editing a form. So there are inputs. How to disable autoupdate and autosave until I finish typing?

**Reply by filipelima18** [reply]
It looks like this is browser behavior and not something we can immediately fix. I've brought this to the team internally for further exploration, but I don't have an expected timeline for a fix.
